src/index.js

```diff
@@ -60,6 +61,7 @@ export { default as ContainerDeprecated } from './layout/containerDeprecated';
 export { default as Content } from './layout/content';
 export { default as Grid } from './layout/grid';
 export { default as Page } from './layout/page';
+export { default as PageActionBar } from './layout/page/pageActionBar';
```

Comment:
Why is this export necessary?  Just curious?  It's already available as `<Page.ActionBar>`, so this makes it available as `<PageActionBar>` as well, right?